Lab 1/.gitignore

```diff
@@ -0,0 +1,261 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Both `.gitignore` and `.gitattributes` should be only in the root of the repository.
Don't need to repeat these files across all labs' folders.